app/controllers/admin/radios_controller.rb

```diff
@@ -39,12 +39,15 @@ def create
 
   # PATCH/PUT /radios/1
   def update
-    @radio.published_at = published_at_from(
-      params[:radio][:status],
-      Time.zone.parse(
-        datetime_select_to_a(params[:radio], :reserve_time).join,
-      ),
-    )
+    unless @radio.publish? && params[:radio][:status] == "publish"
+      @radio.published_at = published_at_from(
+          params[:radio][:status],
```

Comment:
__[RuboCop]__

Layout/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.


<!-- Comment by [SideCI](https://sideci.com) -->